plugins/pi4j-plugin-linuxfs/src/main/java/com/pi4j/plugin/linuxfs/provider/i2c/LinuxFsI2C.java

```diff
@@ -157,6 +181,105 @@ public int readRegister(int register, byte[] buffer, int offset, int length) {
         });
     }
 
+
+    // this is actually in LinuxFile but not visible,   how to properly handle this ???
+    protected static int getWordSize() {
```

Comment:
Maybe put this method in a helper class somewhere?

Reply:
That is originally in the linuxFile class and not visible from this class.  I will find a utility home for this and have the new and the old usage changed

Reply:
com.pi4j.plugin.linuxfs.util;    added the util here.  As this is in a different package than its earlier usage I cannot use  the util from that old package .